packages/webapp/src/components/Form/Checkbox/index.jsx

```diff
@@ -15,6 +15,7 @@ const Checkbox = ({
   hookFormRegister,
   errors,
   sm,
+  tooltipContent = '',
```

Comment:
Can this be `tooltipContent = undefined` so that the line 45 can simply be `tooltipContent={tooltipContent}`?
I'm just wondering.

Reply:
Good question. I think it is more succinct to do it the way you are suggesting. 

To be honest I was mostly copying the PurePlantingDate component way of doing it. 

For some reason setting anything as undefined makes me uncomfortable for some undefinable reason.

Are you good if also switch it to null?

Reply:
Yes :)